Makefile

```
@@ -546,11 +546,12 @@ helm-template-%: clean-charts charts-integration
 	./hack/bin/helm-template.sh $(*)
 
 # Ask the security team for the `DEPENDENCY_TRACK_API_KEY` (if you need it)
+# changing the directory is necessary because of some quirkiness of how 
+# runhaskell / ghci behaves (it doesn't find modules that aren't in the same 
+# directory as the script that is being executed)
 .PHONY: upload-bombon
 upload-bombon:
-	nix build -f nix wireServer.allLocalPackagesBom -o "bill-of-materials.$(HELM_SEMVER).json"
-	./hack/bin/bombon.hs -- \
-		--bom-filepath "./bill-of-materials.$(HELM_SEMVER).json" \
+	cd ./hack/bin	&& ./bombon.hs -- \
```

```suggestion
	cd ./hack/bin && ./bombon.hs -- \
```
`<TAB>` isn't needed here.

---

i have no idea, how a `<TAB>` got here, I don't even have this character in my editor. (tabs get replaced by two spaces...)